### PR TITLE
Status issues

### PR DESCRIPTION
**Pull request by lecriste**
Warn user of poor resources usage and fix `crab status --summary`.

### CODE REVIEW

File: src/python/CRABClient/Commands/status.py [hunk 1]
```diff
@@ -355,7 +359,7 @@ def checkUserJobids(self, statusCacheInfo, userJobids):
         if wrongJobIds:
             raise ConfigurationException("The following jobids were not found in the task: %s" % wrongJobIds)
 
-    def printDetails(self, dictresult, jobids=None, quiet=False):
+    def printDetails(self, dictresult, jobids=None, quiet=False, maxMemory=2000, maxJobRuntime=1315):
```
Comment (belforte) [inline src/python/CRABClient/Commands/status.py]: default valules should not be hardwired in two places. Need to pick maxMemory and maxJobRuntime via e.g.
`from CRABClient.ClientMapping import parametersMapping, getParamDefaultValue`

**Comment by belforte** [comment]
I merged to facilitate testing, but this PR needs a bit of work.
1)
See comment:
https://github.com/dmwm/CRABClient/pull/4748#pullrequestreview-97787394
2)
what exactly is the issue with site history ? Looks like you are trying to protect the code against
using an empty list, but the problem is that site must be known for a job which ran.

**Comment by belforte** [comment]
@lecriste in the commit comment please refer to which issue you are fixing, so that github will close automatically when the commit is merged in the master (besides having the cross refs from code to issue)

**Comment by lecriste** [comment]
I agree with 1), wanted to do that way already but did not find the functions, thanks for pointing me.
2) The Unknown site problem was due to a [misconfiguration](https://github.com/dmwm/CRABClient/issues/4739#issuecomment-366291526) in the puppet repo. The protections I added fix this [issue](https://github.com/dmwm/CRABClient/issues/4732) in case the site is unknown for any reason and help the debugging.

**Pull request merged**